.eslintrc

```diff
@@ -1,13 +1,9 @@
 {
   "extends": [
-    "react-app",
-    "airbnb",
```

Comment:
Let's just add an issue to include this again.

Reply:
I could add them back.